.gitignore

```diff
@@ -5,7 +5,7 @@ node_modules/
 *.egg-info/
 .ipynb_checkpoints
 *.tsbuildinfo
-xellgrid/qgrid2/labextension
+xellgrid/xellgrid/labextension
```

Comment:
A general comment - can you check if the backend is linked up 100% to the front end? May be a small change to the menu in new code base will confirm. I did some research and it looks like sometimes the backend still pointing to the previous implementation of the front end even after the front end naming change.

Reply:
i see what's going on. i forgot to change widget module name and view name. when I did a fresh build the grid is not linked to backend. will fix in next MR